src/events.psm1

```diff
@@ -468,7 +468,7 @@ function New-SafeguardEventSubscription
         if ($PSBoundParameters.ContainsKey("UserToSubscribe"))
         {
             $local:UserEmailAddress = (Get-SafeguardUser -AccessToken $AccessToken -Appliance $Appliance -Insecure:$Insecure -UserToGet $UserToSubscribe).EmailAddress
-            If([string]::IsNullOrWhitespace($local:User.EmailAddress))
```

Comment:
@Goelrachit 
Did you mean to use the variable from the line above here?  I'm proposing that you did.

Reply:
I did not intend to use the variable from above. See the above comment.